examples/simple_xo/src/main.rs

```diff
@@ -0,0 +1,246 @@
+use sawtooth_xo::handler::XoTransactionHandler;
```

Comment:
Please add copyright headers to all the files (following the other files in the repo). Don't just blindly copy the 'Copyright' line though, make sure it's accurate. If you feel this is sufficiently derived from other code, include multiple 'Copyright' lines (the one(s) from the other code plus one for (presumably) IBM).